src/Ztm.Zcoin.Synchronization/IBlocksStorage.cs

```diff
@@ -10,6 +10,8 @@ public interface IBlocksStorage
 
         Task<(Block block, int height)> GetAsync(uint256 hash, CancellationToken cancellationToken);
 
+        Task<(Block block, int height)> GetByTransactionAsync(uint256 hash, CancellationToken cancellationToken);
```

Comment:
A transaction can be stored more than one blocks. So we have two options here:

1. Make it right.
2. Remove it.

Reply:
Removed